src/components/load-button/load-button.jsx

```diff
@@ -26,6 +26,7 @@ const LoadButtonComponent = ({
             </ButtonComponent>
             <input
                 disabled
+                accept=".sb,.sb2,.sb3,.zip"
```

Comment:
I'd suggest removing `.zip` and adding `.sbx` (ScratchX project)

Reply:
I removed .zip and .sb (since we can't support it yet). Projects now should save as .sb3 as well. I'm leaving .sbx out since they probably can't be fully supported, but we can always reconsider this later.